### PR TITLE
Add dooit-extras

### PR DESCRIPTION
**Pull request by claudiushaag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by claudiushaag** [comment]
@conda-forge/help-python , ready for review!

File: recipes/dooit-extras/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+context:
+  name: dooit-extras
+  version: "0.2.0"
+  python_min: "3.9"
```
Comment (ocefpaf) [inline recipes/dooit-extras/recipe.yaml:4]: You need that only if your min Python is higher than conda-forge's min. We do try to stay in sync with https://devguide.python.org/versions/  

```suggestion
```

**Comment by claudiushaag** [comment]
@ocefpaf, I changed the recipe as you suggested, but it seems as rattler-build does not have access to `python_min`. Could this be related to [#2351 ](https://github.com/conda-forge/conda-forge.github.io/issues/2351) or #28445, where problems with `python_min` are also present?

I would now tend to keep the context and keep my eye on how this problem is tackled.

**Comment by ocefpaf** [comment]
@wolfv what is the recommendation here for the new recipe format?

**Pull request merged**